### ISSUE TITLE
pom.xml:296-302: Dependency-violations is broken. As it...

### ISSUE THREAD

**Issue by davvd** [teamed/qulice#365]
Puzzle `281-2350cdf0` in `qulice-maven-plugin/pom.xml:296-302` has to be resolved: Dependency-violations is broken. As it uses maven-dependency-analyzer 1.4. The latter depends on ASM3. We can't add ASM3 in classpath as findbugs and PMD depends on ASM5. Find a way to upgrade maven-dependency-analyzer to use ASM5. After fix uncomment next line

dependency-violations/pom.xml

If you have any technical questions, don't ask me, submit new tickets instead

**Comment by davvd** [comment]
@pandreev please go ahead, it's your task now, keep [this](http://www.xdsd.org/2014/04/17/how-xdsd-is-different.html) in mind, and don't hesitate to ask any technical questions you may have

**30 mins** is the budget of the task. This is exactly how much will be paid when the problem is [solved](http://www.xdsd.org/2014/04/15/definition-of-done.html) (no matter how much time you will actually spend). See [this](http://www.xdsd.org/2014/04/17/how-xdsd-is-different.html) for more information

**Comment by davvd** [comment]
@pandreev the ticket is with you for 11 days already. If it's not finished in the next 48 hours, it will be re-assigned to someone else, see [No Obligations](http://www.xdsd.org/2014/04/13/no-obligations-principle.html) principle, [this article](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html) should help if you're stuck; as a reminder, don't forget to remove the `@todo` puzzle from the source code; added -30 to your rating, now it is equal to -240

**Comment by davvd** [comment]
@pandreev you've been working with this task for too long. I'm sorry, but I'll assign someone else. Please stop working with it. See our [no obligations](http://www.xdsd.org/2014/04/13/no-obligations-principle.html) principle... added -60 to your rating, now it is equal to -330

**Comment by krzyk** [comment]
@davvd this is postponed

**Comment by davvd** [comment]
> @davvd this is postponed

@krzyk OK, I put "postponed" label here

**Comment by davvd** [comment]
> @davvd this is postponed

@krzyk I will assign somebody else to this issue

**Comment by krzyk** [comment]
@davvd this is not postponed

**Comment by krzyk** [comment]
@davvd assign @krzyk

**Comment by krzyk** [comment]
@davvd please see PR #643

**Comment by davvd** [comment]
> @davvd this is not postponed

@krzyk got it, "postponed" tag removed from here

**Comment by davvd** [comment]
> @davvd assign @krzyk

@krzyk OK @krzyk please proceed, this task is yours

**Comment by davvd** [comment]
> @davvd please see PR #643 

@krzyk will check it soon, thanks

**Comment by davvd** [comment]
@krzyk Thanks all, closed

**Comment by davvd** [comment]
@krzyk I just added **1 hour** to your account, many thanks for your contribution (75677134).. 3 hours and 13 mins spent here; there is a bonus for fast delivery (m=193); +60 to your rating, your total score is [+4359](http://www.netbout.com/b/35035?open=rating)
